code/part-one/blockchain.js

```diff
@@ -1,27 +1,33 @@
 let SHA512 = require('crypto-js/sha512');
```

Comment:
Probably better to use Node's `crypto` library rather than pulling in a new dependency. It's what I used in `part-two`:
https://github.com/hyperledger/education-cryptomoji/blob/master/code/part-two/processor/utils/helpers.js#L8

Whichever approach we go with, I definitely think it should be consistent across both part-one and part-two.

Reply:
Integrated helpers into part one as well